8.0/BlazorWebAssemblyStandaloneWithIdentity/Backend/SeedData.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend;
+
+public class SeedData
+{
+    public static async void Initialize(IServiceProvider serviceProvider)
```

Comment:
Never use `async void` in an ASP.NET Core application except sometimes for event handlers, but even then, it's usually wrong. https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void
 
```suggestion
    public static async Task InitializeAsync(IServiceProvider serviceProvider)
```

Reply:
***DOE! BART!*** 😖 ... Yes ... and I knew this. I think this was something that I stole from a web resource setting this up and didn't catch it when I was composing this code. Thanks for setting it straight and reminding me to ***be more careful***.